packages/theme-check-docs-updater/src/themeLiquidDocsDownloader.ts

```diff
@@ -1,44 +1,76 @@
-import fs from 'node:fs/promises';
+import { Mode } from '@shopify/theme-check-common';
+import envPaths from 'env-paths';
 import fetch from 'node-fetch';
+import fs from 'node:fs/promises';
 import path from 'node:path';
 
+const paths = envPaths('theme-liquid-docs');
+export const root = paths.cache;
+
+export const ThemeLiquidDocsRoot =
+  'https://raw.githubusercontent.com/Shopify/theme-liquid-docs/main';
+export const ThemeLiquidDocsSchemaRoot = `${ThemeLiquidDocsRoot}/schemas`;
+
 export type Resource = (typeof Resources)[number];
 export const Resources = [
   'filters',
   'objects',
   'tags',
-  'section_schema',
-  'translations_schema',
   'shopify_system_translations',
+  'manifest_theme',
+  'manifest_theme_app_extension',
```

Comment:
Rather than hardcoded schema URLs, we have hardcoded manifests URLs. This means we can change the manifests without requiring new releases.